sopel/modules/wikipedia.py

```diff
@@ -2,11 +2,13 @@
 # Copyright 2013 Elsie Powell - embolalia.com
 # Licensed under the Eiffel Forum License 2.
 from __future__ import unicode_literals, absolute_import, print_function, division
-from sopel import web, tools
+import requests
+from sopel import tools
 from sopel.config.types import StaticSection, ValidatedAttribute
 from sopel.module import NOLIMIT, commands, example, rule
 import json
 import re
+import bs4
```

Comment:
`bs4` needs to be added to `requirements.txt`, it's not part of the Python standard library.

But ideally, another parsing method that doesn't add another requirement would be great.